Microsoft.HealthVault.Fhir/Transformers/ObservationToBloodPressure.cs

```diff
@@ -19,8 +19,11 @@ internal static class ObservationToBloodPressure
         internal static BloodPressure ToBloodPressure(this Observation observation)
         {
             var bloodPressure = observation.ToThingBase<BloodPressure>();
-            
-            bloodPressure.When = ObservationToHealthVault.GetHealthVaultTimeFromEffectiveDate(observation.Effective);
+
+            if (observation.Effective == null)
```

Comment:
Close/open brackets. Also, if we are throwing an exception a more descriptive exception is better: "Effective field with a valid Effectiva date on observation is required". Although I am not sure why we should throw an exception here, what made you modify this?